p2p/protocols/protocol.go

```diff
@@ -322,23 +339,40 @@ func (p *Peer) handleIncoming(handle func(ctx context.Context, msg interface{})
 		return errorf(ErrDecode, "<= %v: %v", msg, err)
 	}
 
-	// if the accounting hook is set, call it
+	// if the accounting hook is set, do accounting logic
 	if p.spec.Hook != nil {
-		err := p.spec.Hook.Receive(p, uint32(len(msgBytes)), val)
+
```

Comment:
Just a warning, a potential conflict with https://github.com/ethersphere/swarm/pull/2018, but ti should be easy to resolve, whichever PR gets merged first. @pradovic

Reply:
No problem, thanks for the heads-up!

Reply:
Thanks for the heads up